release.md

```diff
@@ -100,6 +100,7 @@ Move the release from RC folder to the dist folder. This will begin the file syn
 ....
 enter your apache password
 ....
+> svn delete -m "remove old version" https://dist.apache.org/repos/dist/dev/skywalking/kong/x.y.z
```

Comment:
You should not do this here. Because your website is targeting the mirror and download links.

Reply:
You have to make sure all things ready and remove the old release, and it is not x.y.z, it is the previous release version.

Reply:
I am sorry for my incorrect title and comment. I am going to delete dev directory once the packages had moved to release directory.

Reply:
Zhenxu's point is, removing the previous release. Such as removing 0.1.0 once the 0.2.0 is released. Not the thing in dev.

Reply:
` svn mv ` already removed the package in the dev folder.

Reply:
Got it. Fixed